File: ui/src/app/service-request/servicerequest.ts
export class ServiceRequest {
  id: number;
  name: string;
  phoneNumber: string
  emailAddress: string
  ccc: boolean;
  lis: boolean;
  bad: boolean;
  aw: boolean;
  lh: boolean;
  mp: boolean;
  laundry: boolean;
  se: boolean;
  mr: boolean;
  lt: boolean;
  ccg: boolean;
  ae: boolean;
  cpp: boolean;
  prlc: boolean;
  startDate: Date;
  endDate: Date;
  startTime: Date;
  endTime: Date;
  requestDate: Date;
  comment: string;

   static toArray(jsons: any[]): ServiceRequest[] {
      const serviceRequests: ServiceRequest[] = [];
      if (jsons != null) {
          for (const json of jsons) {
              serviceRequests.push(new ServiceRequest(json));
          }
      }
      return serviceRequests;
  }

  constructor(json?: any) {
    if (json != null) {
      this.id = json.id;
      this.name = json.name;
      this.phoneNumber = json.phoneNumber;
      this.emailAddress = json.emailAddress;
      this.ccc = json.ccc;
      this.lis = json.lis;
      this.bad = json.bad;
      this.aw = json.aw;
      this.lh = json.lh;
      this.mp = json.mp;
      this.laundry = json.laundry;
      this.se = json.se;
      this.mr = json.mr;
      this.lt = json.lt;
      this.ccg = json.ccg;
      this.ae = json.ae;
      this.cpp = json.cpp;
      this.prlc = json.prlc;
      this.startDate = new Date(json.startDate);
      this.endDate = new Date(json.endDate);
      this.startTime = new Date(json.startTime);
      this.endTime = new Date(json.endTime);
      this.requestDate = new Date (json.requestDate);
      this.comment = json.comment;
    }
  }
}

File: src/main/java/com/ss/schedulesys/domain/ServiceRequest.java
package com.ss.schedulesys.domain;

import static javax.persistence.GenerationType.IDENTITY;

import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;
import javax.validation.constraints.NotNull;

import org.hibernate.validator.constraints.Email;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "service_request", catalog = "schedulesys_db")
public class ServiceRequest implements java.io.Serializable {

	private static final long serialVersionUID = -1864422494742504917L;

	@Id
	@GeneratedValue(strategy = IDENTITY)
	@Column(name = "id", unique = true, nullable = false)
	private Long id;
	
	@NotNull
	@Column(name = "name", nullable = false)
	private String name;
	
	@NotNull
	@Column(name = "phone_number", nullable = false)
	private String phoneNumber;
	
	@Email
	@Column(name = "email_address", nullable = false)
	private String emailAddress;
	
	@Column(name = "ccc")
	private boolean ccc;
	
	@Column(name = "lis")
	private boolean lis;
	
	@Column(name = "bad")
	private boolean bad;
	
	@Column(name = "aw")
	private boolean aw;
	
	@Column(name = "lh")
	private boolean lh;
	
	@Column(name = "mp")
	private boolean mp;
	
	@Column(name = "laundry")
	private boolean laundry;
	
	@Column(name = "se")
	private boolean se;
	
	@Column(name = "mr")
	private boolean mr;
	
	@Column(name = "lt")
	private boolean lt;
	
	@Column(name = "ccg")
	private boolean ccg;
	
	@Column(name = "ae")
	private boolean ae;
	
	@Column(name = "cpp")
	private boolean cpp;
	
	@Column(name = "prlc")
	private boolean prlc;

	@NotNull
	@Temporal(TemporalType.TIMESTAMP)
	@Column(name = "start_date", nullable = false)
	private Date startDate;

	@NotNull
	@Temporal(TemporalType.TIMESTAMP)
	@Column(name = "end_date", nullable = false)
	private Date endDate;
	
	@NotNull
	@Temporal(TemporalType.TIMESTAMP)
	@Column(name = "start_time", nullable = false)
	private Date startTime;
	
	@NotNull
	@Temporal(TemporalType.TIMESTAMP)
	@Column(name = "end_time", nullable = false)
	private Date endTime;
	
	@NotNull
	@Temporal(TemporalType.TIMESTAMP)
	@Column(name = "request_date", nullable = false)
	private Date requestDate;
	
	@Column(name = "comment")
	private String comment;
}